poetry/masonry/builders/builder.py

```diff
@@ -146,8 +146,9 @@ def find_files_to_add(self, exclude_build=True):  # type: (bool) -> list
         )
         to_add.append(Path("pyproject.toml"))
 
-        # If a license file exists, add it
-        for license_file in self._path.glob("LICENSE*"):
```

Comment:
I think we should keep this line if there are no `license_files` specified in the `pyproject.toml` file, so that people relying on the current behavior are not surprised and published packages are not suddenly missing license files.

Reply:
@sdispater - I agree this should remain the "default path" and I made sure to retain this behaviour as part of my changes. This functionality was consolidated into the package setup stuff (which is now in `factory.py`) and is currently being discussed above: https://github.com/sdispater/poetry/pull/1367#discussion_r328299477 & https://github.com/sdispater/poetry/pull/1367#discussion_r328491317

The specific lines for automatically adding `LICENSE*` and `COPYING*` is in `factory.py` here:
- https://github.com/techdragon/poetry/blob/issue-1350-license-metadata-improvements/poetry/factory.py#L84-L92

And I also added a test case for this existing/default behaviour when no `license_files` have been specified. 😃https://github.com/techdragon/poetry/blob/issue-1350-license-metadata-improvements/tests/masonry/builders/test_complete.py#L394-L416

Reply:
@sdispater Since the functionality is retained as desired, but powered by new code in a different place, if you don't mind, id like to mark this as resolved. 😄